tests/Integration/SDK/SpanLimitsBuilderTest.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenTelemetry\Tests\Integration\SDK\Trace;
+
+use AssertWell\PHPUnitGlobalState\EnvironmentVariables;
+use OpenTelemetry\SDK\Trace\SpanLimitsBuilder;
+use PHPUnit\Framework\TestCase;
+
+/**
+ * @covers OpenTelemetry\SDK\Trace\SpanLimitsBuilder
+ */
+class SpanLimitsBuilderTest extends TestCase
+{
+    use EnvironmentVariables;
```

Comment:
if you use this trait (or test for anything global like env vars) you should restore the state the state the process was, either before or after any test in that class.

PHPunit test are usually executed in one process (unless configured differently), however for every test in a class a new instance of this test class is created. 
The method `setUp` is executed before every test, the method `tearDown` after every test.
You can see how to reset the env vars using this trait here:
https://github.com/tidal/opentelemetry-php/blob/main/tests/Unit/Contrib/OTLPGrpcExporterTest.php#L30